client/blocks/product-purchase-features/product-purchase-features-list/jetpack-backup-security.jsx

```diff
@@ -21,8 +21,8 @@ export default localize( ( { translate } ) => {
 				description={ translate(
 					'Your site is being securely backed up and scanned with real-time sync.'
 				) }
-				buttonText={ translate( 'Visit security dashboard' ) }
-				href="https://dashboard.vaultpress.com/"
+				buttonText={ translate( 'Visit Activity Log' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'View Activity Log' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).